index.html

```diff
@@ -2682,9 +2686,10 @@ <h3>
         </h3>
 
         <p>
-The <a>DID document</a> MUST be serialized as a JSON document, with one
-additional requirement: The <a>DID document</a> MUST include the
-<code>@context</code> property.
+The <a>DID document</a> MUST be serialized according to the
+<a href="#production">production rules for JSON</a>, with one additional
+requirement: The <a>DID document</a> MUST include the <code>@context</code>
+property.
```

Comment:
Given the discussion around terminology, we probably don't want to call this a "property" anymore, or at the very least not just "property" without qualifiers.

Reply:
Agree with "at the very least not just 'property' without qualifiers. That discussion is happening in an issue right now (can't remember the number off of the top of my head). Once we bike shed that name, the Editors will go through the spec and fix all occurences.

Reply:
Given that we don't know when that discussion is going to be resolved, would you object to making that change in a future PR?

Reply:
If it's tracked in a separate issue that's fine, I just don't want that detail to get lost when we move forward.

Reply:
Tracked in https://github.com/w3c/did-core/issues/463